README.md

```diff
@@ -51,41 +58,108 @@ pip install -r requirements/base.txt
+| DFM       | ResNet-18          |   0.894   |   0.864   |   0.558   |   0.945   |   0.984   |   0.946   |   0.994   | **0.913** |   0.871   |   0.979   |   0.941   |   0.838   |   0.761   |    0.95    |   0.911    |   0.949   |
+| DFM       | Wide ResNet-50     |   0.891   |   0.978   |   0.540   |   0.979   |   0.977   |   0.974   |   0.990   |   0.891   |   0.931   |   0.947   |   0.839   |   0.809   |   0.700   |   0.911    |   0.915    |   0.981   |
+| DFKDE     | ResNet-18          |   0.762   |   0.646   |   0.577   |   0.669   |   0.965   |   0.863   |   0.951   |   0.751   |   0.698   |   0.806   |   0.729   |   0.607   |   0.694   |   0.767    |   0.839    |   0.866   |
+| DFKDE     | Wide ResNet-50     |   0.774   |   0.708   |   0.422   |   0.905   |   0.959   |   0.903   |   0.936   |   0.746   |   0.853   |   0.736   |   0.687   |   0.749   |   0.574   |   0.697    |   0.843    |   0.892   |
+
+### Pixel-Level AUC
+
+| Model     |                    |    Avg    |  Carpet   |   Grid    |  Leather  |   Tile    |   Wood    |  Bottle   |   Cable   |  Capsule  | Hazelnut  | Metal Nut |   Pill    |   Screw   | Toothbrush | Transistor |  Zipper   |
+| --------- | ------------------ | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | :-------: | :--------: | :--------: | :-------: |
+| STFPM     | ResNet-18          |   0.951   |   0.986   |   0.988   |   0.991   |   0.946   |   0.949   |   0.971   |   0.898   |   0.962   |   0.981   |   0.942   |   0.878   |   0.983   |   0.983    |   0.838    |   0.972   |
+| STFPM     | Wide ResNet-50     |   0.903   |   0.987   | **0.989** |   0.980   | **0.966** |   0.956   |   0.966   |   0.913   |   0.956   |   0.974   |   0.961   |   0.946   | **0.988** |   0.178    |   0.807    |   0.980   |
+| PatchCore | ResNet-18          |   0.935   |   0.979   |   0.843   |   0.989   |   0.934   |   0.925   |   0.956   |   0.923   |   0.942   |   0.967   |   0.913   |   0.931   |   0.924   |   0.958    |   0.881    |   0.954   |
+| PatchCore | Wide ResNet-50     |   0.955   |   0.988   |   0.903   |   0.990   |   0.957   |   0.936   |   0.972   |   0.950   |   0.968   |   0.974   |   0.960   |   0.948   |   0.917   |   0.969    |   0.913    |   0.976   |
+| PaDiM     | ResNet-18          |   0.968   |   0.984   |   0.918   | **0.994** |   0.934   |   0.947   |   0.983   |   0.965   |   0.984   |   0.978   |   0.970   |   0.957   |   0.978   |   0.988    |   0.968    |   0.979   |
+| **PaDiM** | **Wide ResNet-50** | **0.979** | **0.991** |   0.970   |   0.993   |   0.955   | **0.957** | **0.985** | **0.970** | **0.988** | **0.985** | **0.982** | **0.966** | **0.988** | **0.991**  | **0.976**  | **0.986** |
+
+### Balanced Accuracy Score
```

Comment:
We don't support the balanced accuracy score anymore since the Github migration. It would be better to leave it out of this list.